syntax = "proto2";

message response_msg2
{
	required string info = 1;
}